src/com/videojs/providers/HTTPVideoProvider.as

```diff
@@ -205,16 +205,14 @@ package com.videojs.providers{
             appendBuffer(FLV_HEADER);
         }
 
-        public function get buffered():Number{
-            // _src.path == null when in data generation mode
-            if(_ns && _src.path == null)
-            {
-                return _startOffset + _ns.bufferLength + _ns.time;
-            } else if(duration > 0){
-                return (_ns.bytesLoaded / _ns.bytesTotal) * duration;
```

Comment:
it looks like you eliminated special casing around not being in data generation mode and also not having a valid duration before calling this.  Will this return similar results as it did previously in reguler VOD mode and prior to having a duration?  I would assume that the buffer information will not be available prior to having a duration so that's ok.  But what about the VOD values -- is backBufferLength to bufferLength available for VOD HTTP netstreams?